site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1077,15 +1075,10 @@ components:
             }]
       MetaPasswordExample:
          value:
-           [{
-               auth-id: sensor1,
-               type: hashed-password,
-               secrets: [{
-                  "id": "349556ea-4902-47c7-beb0-1009ab693fb4",
-                  "not-after": "2027-12-24T19:00:00Z",
-                  "pwd-plain": "",
-                  "pwd-hash": "",
-                  "salt": "",
-                  "hash-function": ""
-               }]
+            auth-id: sensor1
+            type: hashed-password
+            secrets: [{
+               "id": "349556ea-4902-47c7-beb0-1009ab693fb4",
+               "not-after": "2027-12-24T19:00:00Z",
+               "pwd-plain": ""
```

Comment:
what is this property good for in this context? Isn't this the response to a *get credentials* request?

Reply:
the ID property of the secret is provided here so the user can keep track of the different secrets without handing over the actual values. I also thought the `id` property could be provided in the payload to a `PUT` request on the management API if the user need to update this specific secret. (without creating another one, or submitting all of the other credentials, but that is not implemented yet) .

Reply:
I actually meant the empty *pwd-plain* property ...
So, if I get it right, then the *id* property currently has no purpose at all because it is not considered when adding/updating credentials, right?

Reply:
Indeed, the `pwd-plain`  does not have any sense here. removed.
the id field still have a use IMHO. When the user set some credentials, he can "verify" they are in the registry with receiving an id for each password. I think if I don't know hono very well I would be confused if I received an empty secrets array after registering a hashed password. With the id field I know the password is there.